test/unit/org/apache/cassandra/cql3/statements/TransactionStatementTest.java

```diff
@@ -371,23 +376,38 @@ public void shouldRejectLetSelectWithIncompletePartitionKey()
     @Test
     public void shouldRejectTTL()
     {
-        Mutation.Builder builder = Mutation.builder(tbl5())
-                                           .value("k", 1)
-                                           .value("v", 2)
-                                           .ttl(42);
         for (Mutation.Kind kind : Mutation.Kind.values())
         {
             if (kind == Mutation.Kind.DELETE) continue; // deletes don't support TTL
-            Mutation mutation = builder.kind(kind).build();
+            Mutation mutation;
+            switch (kind)
+            {
+                case INSERT:
+                    mutation = Mutation.insert(tbl5())
+                                       .value(new Symbol("k", Int32Type.instance), Bind.of(1))
+                                       .value(new Symbol("v", Int32Type.instance), Bind.of(2))
+                                       .ttl(42)
+                                       .build();
+                    break;
+                case UPDATE:
+                    mutation = Mutation.update(tbl5())
+                                       .value(new Symbol("k", Int32Type.instance), Bind.of(1))
+                                       .set(new Symbol("v", Int32Type.instance), Bind.of(2))
```

Comment:
this one line is different... and has to be different as `UPDATE` has both `SET` and `WHERE` blocks...  you must be explicit.